odysseus/convertFtbQuests.ts

```diff
@@ -478,7 +479,7 @@ export const convertFtbQuests = async (input: QuestInputFileSystem, output: Ques
 
                 const heraclesQuest: HeraclesQuest = {
                     settings: {
-                        hidden: quest.hide
+                        hidden: quest.invisible ? "COMPLETED" : (quest.hide ? "IN_PROGRESS" : undefined)
```

Comment:
Nested ternary is code smell.

Reply:
suggestions?

Reply:
A variable `hidden` before `heraclesQuest` which is initialized in an if statement.

Reply:
done with the let-if-else pattern in 58e9997